microcosm/config/api.py

```diff
@@ -4,6 +4,12 @@
 """
 from microcosm.config.model import Configuration
 from microcosm.decorators import DEFAULTS
+from microcosm.config.validation import (  # noqa: F401
```

Comment:
I'm not ready to change the top-level `microcosm.api` imports, but I think we should once this stabilizes.